file_system/repository/file_repository.py

```diff
@@ -86,6 +86,6 @@ def filter(cls, queryset=None, path=None, path_in=[], path_regex=None, file_type
         if values_metadata_list:
             values_metadata_query_list = ['metadata__%s' % single_metadata for single_metadata in values_metadata_list ]
```

Comment:
Maybe we can add order_by query parameter as a future improvement so we can have the option to choose how to order elements. But for now this is good!